### ISSUE TITLE
Converting from .pth to .onnx failed on 3d-input. Support 3d-conv?

### ISSUE THREAD

**Issue by Joevaen** [pytorch/pytorch#42911]
## ❓ Questions and Help
Hi,
Below picture is the result of a transformation from pth to onnx who are shown in Neutron:
![image](https://user-images.githubusercontent.com/46133615/90018739-fbe30b80-dcdf-11ea-95b4-66877623689d.png)
Left is pth's architecture and right is onnx's architecture. You noticed that the first layer size of left input is different from the right. In my exprienment, as for the onnx model, the input data flowing from the bottom layer(32x1x3x3x3) which is opppsite from the correct situation (flowing from (320x640x3x3x3)).
My origin network of .pth includes conv3d and the input data is 5-dim. Can the onnx be converted to onnx  on conv3d?

cc @houseroad @spandantiwari @lara-hdr @BowenBao @neginraoof

**Comment by spandantiwari** [comment]
@Joevaen - yes, ONNX has support for 3D conv? 
Is this a duplicate of https://github.com/pytorch/pytorch/issues/42788? If not, then please add repro steps for any issue you are seeing.

**Comment by Joevaen** [comment]
> @Joevaen - yes, ONNX has support for 3D conv?
> Is this a duplicate of #42788? If not, then please add repro steps for any issue you are seeing.

Hi，I am struggling with this issue. Like what  I said above, I converted the pth to onnx failed and the flow is obviously wrong. What should I do to deal with it?

**Comment by Joevaen** [comment]
@zhangguanheng66 
@spandantiwari 
@svenstaro 
@eklitzke 
Could you please help me?

**Comment by spandantiwari** [comment]
@Joevaen - as asked before, could you please add the exact repro steps that lead to the graph on the right.

**Comment by Joevaen** [comment]
```
        from torch2trt import torch2trt
        from torch2trt import tensorrt_converter

        x = torch.zeros([1, 1, 96, 160, 160]).to("cuda")
        model_trt = torch2trt(self.network, [x])
        torch.save(model_trt.state_dict(), 'plugin_trt.pth')

        img = torch.zeros([1, 1, 96, 160, 160]).to("cuda")
        torch.onnx.export(self.network, (img),
                          '/home/qiao/nnUNetFrame/nnUNet/tensorrt/model_final_checkpoint.onnx',
                          input_names=["data"], output_names=["prediction"], verbose=True, opset_version=11,
                          operator_export_type=torch.onnx.OperatorExportTypes.ONNX
                          )
        model = onnx.load(
            '/home/qiao/nnUNetFrame/nnUNet/tensorrt/model_final_checkpoint.onnx')
        model_simp, check = simplify(model)
        assert check, "Simplified ONNX model could not be validated"
        onnx.save(model_simp,
                  '/home/qiao/nnUNetFrame/nnUNet/tensorrt/model_final_checkpoint.onnx')
```
This is my code, I convert .pth to onnx by this code. But I failed. HELP!

**Comment by garymm** [comment]
@Joevaen we need to see the source code that produces `self.network` in order to act on this.
Also I suggest that you try without `torch2trt` and without `simplify`, as I have no idea what those do.
Please try with the latest pytorch nightly version and provide the requested info if you want help.

**Comment by thiagocrepaldi** [comment]
Closing as there is no repro or response in a long time
